samples/python/sqlalchemy-sample/test_create_model.py

```diff
@@ -0,0 +1,23 @@
+""" Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from connect import create_test_engine
+from model import mapper_registry
+
+
+engine = create_test_engine(options="?options=-c spanner.ddl_transaction_mode"
+                                    "=AutocommitExplicitTransaction")
```

Comment:
If you used pytest, create_test_engine() could possibly be a fixture or set of fixtures?

Reply:
See also above.

Our test framework spins up a mocked Cloud Spanner instance and a PGAdapter in-process, and then pass the network addresses and port numbers of those to Python as a command line argument. This makes the test framework reusable across multiple ecosystems. We currently use the same base setup for Python, Go, .NET and NodeJS.